charts/steward/templates/clustertask-jenkinsfile-runner.yaml

```diff
@@ -98,6 +105,8 @@ spec:
       value: '$(inputs.params.PIPELINE_PARAMS_JSON)'
     - name: PIPELINE_LOG_ELASTICSEARCH_INDEX_URL
       value: '$(inputs.params.PIPELINE_LOG_ELASTICSEARCH_INDEX_URL)'
+    - name: PIPELINE_LOG_ELASTICSEARCH_CREDENTIAL
+      value: '$(inputs.params.PIPELINE_LOG_ELASTICSEARCH_CREDENTIAL)'
```

Comment:
Is it OK with `inputs.`? See https://github.com/SAP/stewardci-core/pull/173

Reply:
We must be careful when merging.
We MUST NOT mix 'inputs.params.X' with 'params.Y' variable names.

Reply:
I only changed it now for `PIPELINE_LOG_ELASTICSEARCH_INDEX_URL` and `PIPELINE_LOG_ELASTICSEARCH_CREDENTIAL`. I guess the rest should be handled by the PR from Andreas #173